packages/pg/test/integration/client/error-handling-tests.js

```diff
@@ -113,7 +112,7 @@ var ensureFuture = function (testClient, done) {
 suite.test('when query is parsing', (done) => {
   var client = createErorrClient()
 
-  var q = client.query({ text: 'CREATE TEMP TABLE boom(age integer); INSERT INTO boom (age) VALUES (28);' })
+  client.query({ text: 'CREATE TEMP TABLE boom(age integer); INSERT INTO boom (age) VALUES (28);' })
```

Comment:
Is this line even necessary?

Reply:
Separate PR to remove this line at https://github.com/brianc/node-postgres/pull/3256

Reply:
Per https://github.com/brianc/node-postgres/pull/3256#issuecomment-2167937757, `boom` table is referenced inside `ensureFuture()`.